google/cloud/pubsub/publisher.h

```diff
@@ -109,10 +109,12 @@ class Publisher {
   /**
    * Publishes a message to this publisher's topic
    *
-   * @par
    * Note that the message maybe be batched based on the Publisher's
    * configuration, the message may not be immediately sent.
    *
+   * @par Example
+   * @snippet samples.cc publish
```

Comment:
s/publish/publish-custom-attributes/?

Reply:
I was going to add both `publish` and `publish-custom-attributes` to this header, and then I realized it was too much, so I just left `publish` there. Happy to split to a separate PR.